slash/plugins/builtin/notifications.py

```diff
@@ -200,8 +211,8 @@ def _get_message(self):
         else:
             session_info = 'Session ID: {session_id}'
         short_message = "{results_summary}\n\n" + session_info
-        title = "Slash session in {host_name} ended {result} (duration: {duration})"
```

Comment:
@ayalash I thought about it too, but I'm not sure the duration is important enough for the subject. One can always just click the email to see the details...